src/Yardarm.CommandLine/inheritanceOnlyTestSpec.json

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
I'd prefer it if a simple example of the use case were adding to the existing yaml file, rather than adding a whole additional schema file.